examples/plotting/file/tap.py

```diff
@@ -14,20 +14,11 @@
     "#%02x%02x%02x" % (int(r), int(g), 150) for r, g in zip(50+2*x, 30+2*y)
 ]
 
-source = ColumnDataSource(data=dict(
-    x=x,
-    y=y,
-    radius=radii,
-    colors=colors,
-    foo=list(itertools.permutations("abcdef"))[:N],
-    bar=np.random.normal(size=N),
```

Comment:
Those two shouldn't have been removed. They show that you can have extra fields besides what's allowed by the glyph. Though it would be nice to extend this to actually take advantage of those fields, e.g. in a hover tool or a callback.